tests/kafkatest/services/trogdor/degraded_network_fault_spec.py

```diff
@@ -23,7 +23,7 @@ class DegradedNetworkFaultSpec(TaskSpec):
     Degrades the network so that traffic on a subset of nodes has higher latency
     """
 
-    def __init__(self, start_ms, duration_ms, node_specs):
+    def __init__(self, start_ms, duration_ms, node_specs={}):
         """
         Create a new NetworkDegradeFaultSpec.
 
```

Comment:
the comment below don't match the actual constructor parameters.  It lists "node_latencies" and "network_device" which don't exist as parameters, and omits node_specs.